packages/manager/src/features/CloudPulse/Alerts/AlertsLanding/AlertsDefinitionLanding.tsx

```diff
@@ -19,6 +19,7 @@ export const AlertDefinitionLanding = () => {
       <Route exact path={`${url}/edit/:serviceType/:alertId`}>
         <EditAlertLanding />
       </Route>
+      <Redirect from="*" to={`${url}`} />
```

Comment:
Nit: We don't need to use ` and $ to access variable, directly we can pass the variable.
```suggestion
      <Redirect from="*" to={url} />
```